forte/processors/data_augment/utils/utils.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Utility functions
+"""
+
+from os import path
+
+import numpy as np
+
+from texar.torch.data import Embedding, load_glove
+from texar.torch.data import Vocab
+
+__all__ = [
+    "l2_norm",
+    "load_glove_vocab"
+]
+
+
+def l2_norm(word_vecs):
+    r"""Calculate the L2 norm
+
+    Args:
+        word_vecs: 2d numpy array of word vectors.
+
+    Returns:
+        Normalized vectors.
+    """
+    norm = np.sqrt((word_vecs * word_vecs).sum(axis=1))
+    return word_vecs / norm[:, np.newaxis]
+
+
+def load_glove_vocab(glove_filename: str) -> Vocab:
+    r"""Load pre-trained glove vocabulary from file
+
+    Args:
+        glove_filename: Path to the pre-trained glove embeddings file
+
+    Returns:
+        A texar.tf.torch.data.Vocab object
+    """
+    vocab_filename = glove_filename + ".vocab"
+    if not path.isfile(vocab_filename):
+        vocab_list = []
+        with open(glove_filename, "r") as glove_file:
+            for line in glove_file:
+                vec = line.strip().split()
+                if len(vec) == 0:
+                    continue
+                word = vec[0]
+                vocab_list.append(word)
+
+        with open(vocab_filename, "w") as output_file:
+            output_file.write('\n'.join(vocab_list))
+    return Vocab(vocab_filename)
+
+
+def load_glove_embedding(glove_filename: str, emb_dim: int, vocab: Vocab)\
```

Comment:
https://github.com/asyml/texar-pytorch/blob/master/texar/torch/data/embedding.py#L72

Reply:
Yes. In fact, this function is just a helper that wraps the code in the link to initialize an `Embedding`.

Do you suggest to remove this helper function and let the user initialize the `Embedding` themselves?

Reply:
I don't see the need to have such a function. It cannot support a lot of embedding types anyway. The function here is also not really re-used, so just use it in the example

Reply:
Sure. Then I can get rid of `utils.py` entirely. Thanks!